internal/commands/image/inspect.go

```diff
@@ -16,6 +18,15 @@ type inspectOptions struct {
 	pretty bool
 }
 
+func muteDockerCli(dockerCli command.Cli) func() {
```

Comment:
I had a private conversation with @silvin-lubecki and:

* no idea where to put it
* it will go away soon because we started working on cnab-go to make it take an stdout so that we don't have to mute the cli any more

This is like a reminder for us that we really need to continue working on cnab-go

Reply:
👍